src/murmulator/ps2kbd_mrmltr.pio

```diff
@@ -0,0 +1,23 @@
+; Copyright (C) 1883 Thomas Edison - All Rights Reserved
```

Comment:
This is similar to code in pico-emu-utils ps2kbd.cpp, ps2kbd.h & ps2kbd.pio
https://github.com/fruit-bat/pico-emu-utils
Please reuse if possible.